x/harvest/keeper/borrow.go

```diff
@@ -46,6 +46,16 @@ func (k Keeper) Borrow(ctx sdk.Context, borrower sdk.AccAddress, coins sdk.Coins
 	// Update total borrowed amount
 	k.IncrementBorrowedCoins(ctx, coins)
 
+	// TODO: Refactor TotalBorrows store key and BorrowedCoins store key. The logic is currently duplicated.
```

Comment:
What's the difference between `GetBorrowedCoins` and `GetTotalBorrows`? Not sure I have a preference for one or the either, but I don't think we need both.

Reply:
Removed `GetTotalBorrows` in https://github.com/Kava-Labs/kava/pull/720/commits/313c8b845c882457c71ab4d7cb65d93b7f4e6d6a and am just using `GetBorrowedCoins`